update to the latest DS version

mck opened the pull request.

mck committed:
update to the latest DS version

mck committed:
changeset

mck committed:
update packages

six7 reviewed (approved):
Nit: no need to specify tooltip if everytime if it stays the default

six7 reviewed (approved):
Nit: no need to specify tooltip if everytime if it stays the default

mck committed:
Remove redundant tooltip ID from toolbar components

Deleted the unnecessary 'data-tooltip-id' attribute from SettingsToolbarButton and HelpDropdown components to clean up props and prevent potential tooltip conflicts.

mck committed:
fix shadows

mck committed:
update toolbar to container data attribute
